src/UartSerial.cpp

```diff
@@ -156,7 +157,7 @@ UartSerial::begin (
     // Leave all output flags unset
 
     // c_cflag - control modes
-    _tio_config.c_cflag |= CS8;  //TODO: Configuration passed by caller
+    _tio_config.c_cflag |= c_cflags;
```

Comment:
Ok, I looked back over this and there are only 4 POSIX bits we would vary in c_cflags based on the config passed in. They are CSIZE, PARENB, PARODD, and CSTOPB. I feel like the code we have in this commit to simply set them straight away is the easiest to understand. We will not inherit the existing c_cflag value so NOTing out the bits we don't use seems redundant since we end up with something like:
```c++
case SERIAL_5E1:
  c_cflags = CS5 | PARENB; 
  c_cflags &= ~(CSTOPB | PARODD);
  break;
```
We'll never have bits set that we don't expect from the first line and the second is basically a no-op because those bits wouldn't be set anyway. What do you think?